test/modeles/autorisations/autorisationBase.spec.js

```diff
@@ -141,40 +159,72 @@ describe('Une autorisation de base', () => {
   });
 
   describe('sur demande de permission de gestion des contributeurs', () => {
-    it("interdit la gestion si l'autorisation ne provient pas d'un créateur", () => {
-      const autorisationContributeur = new AutorisationContributeur();
+    it('interdit la gestion pour un contributeur', () => {
+      const autorisationContributeur =
+        AutorisationBase.NouvelleAutorisationContributeur();
 
       expect(autorisationContributeur.peutGererContributeurs()).to.be(false);
     });
 
-    it("autorise la gestion si l'autorisation provient d'un créateur", () => {
-      const autorisationCreateur = new AutorisationCreateur();
+    it('autorise la gestion pour un propriétaire', () => {
+      const autorisationCreateur =
+        AutorisationBase.NouvelleAutorisationProprietaire();
 
       expect(autorisationCreateur.peutGererContributeurs()).to.be(true);
     });
   });
 
-  it('connaît ses données à persister', () => {
-    const autorisationContributeur = new AutorisationContributeur({
-      id: 'uuid',
-      idService: '123',
-      idUtilisateur: '999',
-      droits: tousDroitsEnEcriture(),
+  describe('sur demande des données à persister', () => {
+    it('connaît ses données pour une autorisation de contributeur', () => {
+      const autorisationContributeur =
+        AutorisationBase.NouvelleAutorisationContributeur({
+          id: 'uuid',
+          idService: '123',
+          idUtilisateur: '999',
+          droits: tousDroitsEnEcriture(),
+        });
+
+      expect(autorisationContributeur.donneesAPersister()).to.eql({
+        estProprietaire: false,
+        id: 'uuid',
+        idService: '123',
+        idHomologation: '123',
+        idUtilisateur: '999',
+        type: 'contributeur',
+        droits: {
+          CONTACTS: 2,
+          DECRIRE: 2,
+          HOMOLOGUER: 2,
+          RISQUES: 2,
+          SECURISER: 2,
+        },
+      });
     });
 
-    expect(autorisationContributeur.donneesAPersister()).to.eql({
-      id: 'uuid',
-      idService: '123',
-      idHomologation: '123',
-      idUtilisateur: '999',
-      type: 'contributeur',
-      droits: {
-        CONTACTS: 2,
-        DECRIRE: 2,
-        HOMOLOGUER: 2,
-        RISQUES: 2,
-        SECURISER: 2,
-      },
+    it('connaît ses données pour une autorisation de propriétaire', () => {
+      const autorisationProprietaire =
+        AutorisationBase.NouvelleAutorisationProprietaire({
+          id: 'uuid',
+          idService: '123',
+          idUtilisateur: '999',
+          droits: tousDroitsEnEcriture(),
```

Comment:
On pourrait dire que `NouvelleAutorisationProprietaire()` ne **prend pas** les droits en paramètre. Et qu'elle applique toujours `tousDroitsEnEcriture()`. Ça correspond au métier.

Reply:
C'est noté dans la checklist pour le faire plus tard :+1: